README.md

```diff
@@ -29,3 +29,34 @@ mailing list as well.
 
 ### Other useful links
 * [Public group email archive](https://lists.w3.org/Archives/Public/public-vc-wg/)
+
+## Process Overview for VC Data Model Pull Requests
+1. For now, we will focus only on merging new errata PRs into this repository,
+but encourage activity related to new features.
+2. Once a PR is opened, chairs and editors make judgement call on whether
+changes are substantive or editorial.
+```
+Editorial   => mark with "editorial" tag
+               would be merged into branch "v1.1"
+Substantive => mark with "substantial" tag
+    Bugfix       |=> goes into separate branch "v1.2"
+    New Features |=> stays as an open PR?
+```
```

Comment:
I can't suggest what's needed here, because Github doesn't handle indented codefences properly.  Lines 38 through 44 need to be indented with 3 space characters, in order to nest that codeblock under item 2.

Reply:
A definition list might be the way to go. It looks like GitHub doesn't support the extended [Markdown Definition List](https://www.markdownguide.org/extended-syntax/#definition-lists), at least not in comments. HTML in Markdown shouldn't be a problem.